test/data/licenceClientTest.js

```diff
@@ -309,4 +309,64 @@ describe('licenceClient', () => {
       expect(sql).to.include(expectedContents3)
     })
   })
+
+  describe('getLicencesInStageBetweenDates', () => {
+    it('should call query', () => {
+      licencesProxy().getLicencesInStageBetweenDates('stage', 'from', 'upto')
+      expect(queryStub).to.have.callCount(1)
+    })
+
+    it('should pass in the correct sql', () => {
```

Comment:
Curious.. This test seems like it's testing hard coded values from the method and from the looks of it is only giving you assurance that the sql query didn't change... is this the intention?

Reply:
No, that's pretty pointless. Following the pattern matt established in the rest of that test. Will remove, there's nothing dynamic in that query